source/OptimaJet.DWKit.StarterApplication/Controllers/BaseController.cs

```diff
@@ -67,7 +67,7 @@ private async Task<User> FindOrCreateCurrentUser()
                 FamilyName = currentUserContext.FamilyName
             };
 
-            var newEntity = await userService.CreateAsync(newUser);
+            var newEntity = await userService.EntityRepository.CreateAsync(newUser);
```

Comment:
in general, controllers should not touch repositories

Reply:
Chris had wanted to make CreateAsync at the service level return not implemented, which is why the change was made.  Changed this to call a new DirectCreateAsync at the service level which simply calls the service base.GetAsync(id)

Reply:
it should be at the controller level for the user. :-\

Reply:
semantically, it should be a 405 status code, I think

Reply:
Moved change to controller and return a 405 on post to user.  Reversed previous changes so that the base controller calls user service createasync as it did originally.  Added test that verifies failure 405 adding new user.